addons/gsri_opex/functions/fremm/fn_fremmAddCrewQuarters.sqf

```diff
@@ -9,6 +9,8 @@ if(isServer) then {
 	_roomA setDir (getDir _ship + 90);
 	_roomA call BIS_fnc_Destroyer01PosUpdate;
 	_ship setVariable ["GSRI_FREMM_CrewQuartersRoomA", _roomA, true];
+	[_roomA, "A"] call GSRI_fnc_crewAddProps;
```

Comment:
In the previous version you created both rooms then added props, now you're changing instructions order.
Did you test it did not cause havoc to add props to room A before creating room B ?

Reply:
I haven't even thought about it. As far as I can see, this causes absolutely no problem, as long as spawning props for A is done after room A is created. After that all props are attachTo with relative coordinates to their given room, so having room B spawned or not when placing props for A is not relevant.

Seems ok to me \o/